### DISCUSSION TITLE
Is homebrew the best way to install pipenv anymore?

### DISCUSSION THREAD

**Discussion by CarlaAstudillo** [texastribune/data-visuals-guides#26] [General]
In our instructions for [installing pipenv](https://github.com/texastribune/data-visuals-guides/blob/3fc258bca8fcd5226075b78dce22db21d8b849de/computer-setup.md?plain=1#L158), we use homebrew to install it.

However, the [pipenv documetation](https://pipenv.pypa.io/en/latest/installation.html) for installation recommends that we don't use homebrew anymore to install pipenv. They recommend instead to use pip.

<img width="821" alt="Screenshot 2024-12-17 at 3 13 17 PM" src="https://github.com/user-attachments/assets/ce8cc403-f103-460c-bac8-d7f310903c31" />

When I tried to use pip (well, pip3: related to this), it gave me a pretty gnarly error which I'm wondering if it's because we use homebrew to install python.

<img width="1057" alt="Screenshot 2024-12-17 at 3 25 24 PM" src="https://github.com/user-attachments/assets/36b2681c-ac0f-4ad7-8dd4-999f0bcba7d3" />

I just ended up using homebrew to install pipenv and honestly it has not caused me problems!

So should we look into not installing pipenv using homebrew anymore? If so, should we then look into installing python without using homebrew? Or should we just stay the course (it hasn't really caused any problems for me)?

**Comment by csessig86** [comment]
I'm fine still installing `pipenv` with homebrew, since it hasn't caused issues. I do think we should make installing pyenv required and move it up to the python section. With `pyenv` installed, it looks like`pipenv` uses that version, which is what we want:

<img width="580" alt="Screenshot 2025-01-24 at 2 12 09 PM" src="https://github.com/user-attachments/assets/44f67ca1-d8d1-46f0-99c9-0e251f2e6335" />

@reliablerascal I know you installed somewhat recently so let me know if this seems okay with you.
